src/disc/stk/STKConnManager.cpp

```diff
@@ -126,7 +120,7 @@ STKConnManager::getConnectivityMask (const std::string& sub_part_name) const
         int count = m_bulkData->num_connectivity(e,rank);
         for (int irel=0; irel<count; ++irel) {
           for (int cnt=0; cnt<m_idCnt[rank]; ++cnt) {
-            elem_mask[offsets[rank] + cnt] = 1;
+            elem_mask[offsets[rank] + irel*m_idCnt[rank] + cnt] = 1;
```

Comment:
Was this a bug?

Reply:
Yes, but only showed up for a particular case. Namely, when we restricted to a sideset. Currently, that's never the case: we only call getConnectivityMask when restricting the DOF mgr to a nodeset (e.g., bottom for extruded meshes). However, when building an extruded discretization (coming in a follow up PR), I needed to restrict to a side set, and started getting seg faults.